doc/content/lorawan/db-dbm-dbi-dbd/_index.md

```diff
@@ -0,0 +1,100 @@
+
+A power ratio greater than 0 dB is treated as a gain. For example, if an amplifier turns a 2 W signal into a 10 W signal, the power ratio is,
+
+10 log<sub>10</sub> P<sub>out</sub>/P<sub>in</sub>  = 10 log<sub>10</sub> 10/2   = 10 log<sub>10</sub> 5  = 6.9 dB (gain)
+
+A power ratio less than 0 dB is treated as a loss (negative gain or attenuation). For example, if 10 W of power is fed into a cable but only 8 W is measured at the output, the power ratio is,
+
+10 log<sub>10</sub> P<sub>out</sub>/P<sub>in</sub>  = 10 log<sub>10</sub> 8/10 = 10 log<sub>10</sub>(0.8) = -0.9 dB (loss)
+
+The power ratio of 0 dB means there is no gain or a loss.
+
+## dBm (decibel-milliwatt)
+
+If you use the reference input power of 1mW the power ratio can be expressed in dBm (‘m’ stands for milliwatt).
+
+**dBm = 10 log<sub>10</sub>(P<sub>out</sub> / 1 mW)**
+
+By using the above formula, P<sub>out</sub> can be expressed as an absolute value using W or mW as follows.
+
+**P<sub>out</sub> = 10<sup>(dBm/ 10)</sup>**
+
+For example, if a LoRaWAN gateway is rated as 22 dBm how much power does it generate in Watts?
+
+P<sub>out</sub> = 10<sup>(dBm/ 10)</sup>  = 10<sup>(22 / 10)</sup>  = 10<sup>(2.2)</sup> = 158.48 mW = 0.158 W
+
+### Rule of 10s and 3s
+
+The rule of 10s and 3s can be used easily to convert a dBm value to its corresponding absolute power value without using the logarithmic scale.
+
+- 10 dB = x10 (makes output power 10 times as the input power, for example, input=10 W and output=100 W)
+- -10 dB = ÷10 (makes output power 1/10 times as the input power, for example, input=100 W and output=10 W)
+- 3 dB = x2 (doubles the power, for example, input=5 W and output=10 W)
+- -3 dB = ÷2 (halves the power, for example, input=10 W and output=5 W)
+
+Only use 10s and 3s (positive or negaive) to express the given dBm value, for example, 1 dBm can be written as, 10 -3 -3 -3.
+
+Then apply the rule:
+
+1 dB = 10dB -3dB -3dB -3dB = x10 ÷2 ÷2 ÷2 = 1.25 mW
+
+When you write any dBm value in 10s and 3s remember,
+- Only use multiple of 10s and 3s
+- If possible avoid using 3s
+- Never use more than five 3s
+
+Let's take another example:
+
+If a LoRaWAN gateway has 17 dBm, how much power does it generate?
+
+17dB = 10 +10 -3 = x10 x10 ÷2 = 50mW
+
+## dBi (decibel-isotropic)
+
+The gain of an antenna can be measured relative to an isotropic antenna and is expressed in dBi ('i' stands for isotropic). An isotropic antenna is a hypothetical antenna that radiates power uniformly in all directions. The gain of an isotropic antenna is 0 dB which means it has no gain or loss. 
+
+## dBd (decibel-dipole)
+The gain of an antenna can be measured relative to a reference dipole antenna and is expressed in dBd ('d' stands for dipole). A reference dipole antenna offers a fixed 2.15 dB of gain over an isotropic antenna.
+
+The following equation represents the relationship between dBi and dBd:
+
+**dBi = dBd + 2.15**
+
+## Questions:
```

Comment:
These questions are only related to antenna gains, so maybe make all three these last headings `###` and put them under on `## Antenna gains` heading?